CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+or [Hypertrace slack](https://bit.ly/hypertrace-slack) with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+4. You may merge the Pull Request once you have the required approvals. If you face any issues merging the Pull Request, you can request reviewer to merge it. 
+
+## Support Channels
```

Comment:
when pull request is pulled up, this becomes slightly redundant with the first paragraph.  in any case look at some other project CONTRIBUTING files for inspiration and clarification